src/main/java/net/masterthought/cucumber/json/Feature.java

```diff
@@ -31,8 +31,9 @@
     public String getDeviceName() {
         String name = "";
         String[] splitedJsonFile = jsonFile.split("_");
-        if (splitedJsonFile.length > 1)
-            name = splitedJsonFile[0].substring(0, splitedJsonFile[0].length() - 1);
+        if (splitedJsonFile.length > 1){
```

Comment:
extract this magic numbers into separate method and add notes why this magic numbers are valid